packages/cli/src/rpc/explain/navie/navie-remote.ts

```diff
@@ -35,7 +37,47 @@ export default class RemoteNavie extends EventEmitter implements INavie {
         async onRequestContext(data) {
           try {
             if (data.type === 'search') {
-              return await self.contextProvider(data as unknown as Context.ContextRequest);
+              const { version } = data;
+              let contextRequestV2: ContextV2.ContextRequest;
+              // Adapt v1 calls to v2
+              const isVersion1 = !version || version === 1;
+              if (isVersion1) {
+                const contextRequestV1 = data as ContextV1.ContextRequest;
```

Comment:
Here a v1 request is converted to v2 format. This may occur if lib-navie is emitting v1, for example if navie is running on getappmap.com and it hasn't been updated yet.